packages/platforms/accton/x86-64/x86-64-accton-asxvolt16/platform-config/r0/src/python/x86_64_accton_asxvolt16_r0/__init__.py

```diff
@@ -0,0 +1,77 @@
+from onl.platform.base import *
+from onl.platform.accton import *
+
+class OnlPlatform_x86_64_accton_asxvolt16_r0(OnlPlatformAccton,
+                                             OnlPlatformPortConfig_20x100):
+    PLATFORM='x86-64-accton-asxvolt16-r0'
+    MODEL="ASXvOLT16"
+    SYS_OBJECT_ID=".volt.16"
```

Comment:
@roylee123  The SYS_OBJECT_ID is invalid.

Reply:
As RoyLee has replied with modification, please review it